components/proxy/src/main/java/com/hotels/styx/proxy/backends/file/FileBackedBackendServicesRegistry.java

```diff
@@ -90,27 +108,45 @@ public static FileBackedBackendServicesRegistry create(String originsFile) {
         return super.stop();
     }
 
+    FileMonitor monitor() {
```

Comment:
Is this package-private because it is for a test? Let's use the `@VisibleForTesting` annotation as we have done in other classes.

Reply:
That is correct. Will fix it.